packages/devextreme/testing/tests/DevExpress.ui.widgets/selection.test.js

```diff
@@ -2442,6 +2442,58 @@ QUnit.module('onSelectionChanging', {
         assert.strictEqual(selection.isItemSelected(this.data[0]), true, 'isItemSelected returns true');
     });
 
+    QUnit.test('should be called with correct parameters when select all ', function(assert) {
```

Comment:
Don't we really have such a test where we check that parameters are correct? If we have then the most part of this test is a copypaste and i'd rather vote for adding one more assertion to existing test (or check only one new parameter here)